addons/stock/models/stock_move_line.py

```diff
@@ -790,6 +795,35 @@ def get_aggregated_properties(move_line=False, move=False):
 
         return aggregated_move_lines
 
+    def _get_empty_moves(self):
```

Comment:
Not sure to understand... This method is really used ?
The empty moves aren't already returned by `_get_aggregated_product_quantities` in the case we could call `_get_empty_moves` as the `strict` kwarg is only passed for the package ?

Reply:
I don't understand what you mean in your message
This method is useful because we don't print the empty moves (undelivered products) in the package section anymore so we need to print them after

Reply:
The only place where we call `_get_empty_moves`, we also call `_get_aggregated_product_quantities` (without the `strict` kwarg), so it should already return the move lines `_get_empty_moves` could return.
Maybe I miss an usecase where this method is relevant, but for now I think this method is never called because `_get_aggregated_product_quantities` already does its job.

Have you a flow where we actually go  through `_get_empty_moves` ?

Reply:
Ok indeed, I understand better the issue now, thanks !

Reply:
Beside my previous comment, instead of creating a new method (`_get_empty_moves`), can't you reuse the `_get_aggregated_product_quantities` but with an extra kwarg, like you did with `strict`, but to return only empty moves ?